index.js

```diff
@@ -124,11 +124,15 @@ Collector.prototype.flush = function(callback) {
 // @param [Object] value the data
 // @param [Object] tags the tags (optional)
 Collector.prototype.collect = function(seriesName, value, tags) {
```

Comment:
would it make sense to name the argument we accept `fields` instead? I'm trying to figure out which the preferred case should be